<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgRegistrationWindow</class>
 <widget class="QDialog" name="dlgRegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>574</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="btnOk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRef">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the reference image:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRef">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelImages">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the images to register:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>461</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDeformationF">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Display the deformation field</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAlgo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Registration algrotihm: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAlgo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdvancedOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced Options</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>accepted()</signal>
   <receiver>dlgRegistrationWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOk</sender>
   <signal>rejected()</signal>
   <receiver>dlgRegistrationWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
